### DISCUSSION TITLE
Successfully Added Wyze Cam V3 to Synology Surveillance Station But Having Trouble Adding the Doorbell

### DISCUSSION THREAD

**Discussion by kfunluc** [mrlt8/docker-wyze-bridge#846] [Q&A]
Hi guys to start off I'm new to owning a NAS and using docker so I've only been able to take myself this far.

It was easy to add my V3 camera with with RTSP firmware because the Wyze app gives the URL to add to surveillance station but for the doorbell, I've tried a variety of IP addresses as well ports but can't seem to get the camera to add manually.

The Wyze Bridge sees the camera in the docker log but I just can't seem to add it. I also can't seem to launch the web GUI because the port 5000 is being used by Synology DSM.

Thanks in advance for anyone who may have some insight.

**Comment by LEMFC** [comment]
Isn't this app for using the Wyze with native firmware without needing RTSP?

**Reply by mockupp** [reply]
"I've tried a variety of IP addresses" - So are you saying you can't find the IP of your doorbell? Use the "FING" app to scan your network.

**Reply by LEMFC** [reply]
I think you were replying to kfunluc, but can you take a second to answer my question too?

**Reply by mrlt8** [reply]
This uses the same method as the Wyze app to connect and communicate with the cameras, so you do not need any special firmware or modifications on the camera itself.

**Comment by mrlt8** [comment]
See https://github.com/mrlt8/docker-wyze-bridge/issues/402?
